### ISSUE TITLE
Add missing aria-labels

### ISSUE THREAD

**Issue by pheeria** [Lernwerkstatt/site#169]
Every form should have `aria` labels

Check [here](https://wave.webaim.org/report#/https://die-lernwerkstatt.org/) for more details
